src/app/SBE/table/Table.js

```diff
@@ -1,21 +1,21 @@
-import { Box, HStack, Stack, Table, useColorModeValue } from '@chakra-ui/react'
-import { equals } from 'ramda'
-import { useSelector } from 'react-redux'
-
+import { Box, HStack, Table, useColorModeValue } from '@chakra-ui/react'
 import getActions, { getTableActions } from '../utils/get-actions'
+
 import Action from 'app/BE/action'
 import Body from './Body'
 import Download from '../download'
 import Header from './Header'
 import MapSearch from 'app/SBE/display_modes/map_view'
 import Pagination from './Pagination'
-import Title from './Title'
-import { apiConfig } from 'config/get-api-config'
 // import Filters from '../filters'
 import Search from 'app/SBE/search/Search'
+import Title from './Title'
```

Comment:
Across all the file import the files using relative import from src.

Reply:
done